helpers/misc_test.go

```diff
@@ -199,3 +200,89 @@ func (suite *TestMiscSuite) TestBoolPtr() {
 func TestMisc(t *testing.T) {
 	suite.Run(t, new(TestMiscSuite))
 }
+
+func TestMergePathAndValueIntoMap(t *testing.T) {
```

Comment:
add this test to the currently existing test suite in this file

Reply:
I did that. Thanks